haas/server.py

```diff
@@ -43,14 +45,13 @@ def stop_orphan_consoles():
     These may exist if HaaS was shut down uncleanly.
     """
     # Stop all orphan console logging processes on startup
-    db = model.Session()
-    nodes = db.query(model.Node).all()
+    nodes = db.session.query(model.Node).all()
```

Comment:
Quoting henn (2016-04-01 00:40:21)

>    Would the pattern of model.Node.query.all() be a little more
>    readable (here and other places), similar to flask/sqlalchemy's
>    quickstart?

I think that's a good idea; I mostly did it this way because it made the
change more mechanical. I'll go through and update it.